src/writeProcessor.ts

```diff
@@ -1,13 +1,14 @@
-import { TypeDefenition } from './typeDefenition';
+import { TypeDefinition } from './typeDefinition';
+import { toTSType } from './utils';
 
 export interface ReferenceResolver {
-    (baseSchema: Schema, ref: string): TypeDefenition;
+    (baseSchema: Schema, ref: string): TypeDefinition;
 }
 
 export class WriteProcessor {
 
     public indentChar = ' ';
-    public indentStep = 4;
+    public indentStep = 2;
```

Comment:
I thought we should almost follow the official [Coding Guideline](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines).
So `Use 4 spaces per indentation.`